README.md

```diff
@@ -12,8 +12,31 @@ if multiple keys are used to encrypt app secrets then this secret should contain
 ### `generate` 
 command will run kustomize build to generate kube resources's yaml strings. it will print this yaml stream to stdout.
 
+You can specify custom SSH keys to be used for fetching remote kustomize bases from private repositories. In order to do that, 
+you will need to set `GIT_SSH_SECRET_NAME` plugin env, it should reference a Secret name that contains one or more SSH keys 
+that provide access to the private repositories that contain these bases. if this env is not set then plugin will 
+only be able to fetch remote bases from open repositories.
+
+To use an SSH key for Kustomize bases, the bases should be defined with the ssh:// scheme in kustomization.yaml and have a 
+`# argocd-strongbox-plugin: key_foobar` comment above it. For example:
```

Comment:
Is now a good time to rename the plugin? Its no longer just does Strongbox

Reply:
i was also thinking about this. any suggestions?